sample/build.gradle

```diff
@@ -0,0 +1,53 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    compileSdkVersion buildConfig.compileSdk
+    buildToolsVersion buildConfig.buildTools
+
+    defaultConfig {
+        applicationId "com.example.sample"
+        minSdkVersion buildConfig.minSdk
+        targetSdkVersion buildConfig.targetSdk
+        versionCode 1
+        versionName releaseConfig.version
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    buildFeatures {
+        viewBinding true
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
+    }
+
+    kotlinOptions {
+        jvmTarget = '11'
+    }
+}
+
+dependencies {
+
+    // TODO - replace with your module names accordingly
+    // Comment this out for testing local deploy
+    implementation project(":libModule1")
+    implementation project(":libModule2")
+
+    // TODO - replace with your module packages accordingly
+    // Uncomment this for testing local deploy
+//    implementation packages.android-libname.libModule1
+//    implementation packages.android-libname.libModule2
+
+    implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'com.android.support.constraint:constraint-layout:2.0.4'
```

Comment:
Can we use AndroidX instead? 😂

Reply:
Sure :D

Reply:
Done as a part of https://github.com/infinum/android-library-template/pull/3 PR